From 9fe38dcb04d89770e12d374880b3115acd9da4f1 Mon Sep 17 00:00:00 2001
From: Christian Fetzer <fetzer.ch@gmail.com>
Date: Tue, 2 Aug 2016 07:54:15 +0200
Subject: [PATCH] [VideoPlayer] Fix build after dropping deinterlacing mode

25718f4ccebd227eaf63f0f8b53f0552f89f01e5 forgot to remove some methods
that still use EDEINTERLACEMODE.
---
 .../VideoPlayer/VideoRenderers/HwDecRender/RendererAML.cpp     | 10 ----------
 .../cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.h |  1 -
 .../VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.cpp |  5 -----
 .../VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.h   |  1 -
 4 files changed, 17 deletions(-)

diff --git a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.cpp b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.cpp
index d0baa27..d8674c8 100644
--- a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.cpp
+++ b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.cpp
@@ -81,16 +81,6 @@ bool CRendererAML::Supports(EINTERLACEMETHOD method)
   return false;
 }
 
-bool CRendererAML::Supports(EDEINTERLACEMODE mode)
-{
-  if(mode == VS_DEINTERLACEMODE_OFF
-  || mode == VS_DEINTERLACEMODE_AUTO
-  || mode == VS_DEINTERLACEMODE_FORCE)
-    return true;
-
-  return false;
-}
-
 bool CRendererAML::Supports(ESCALINGMETHOD method)
 {
   return false;
diff --git a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.h b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.h
index 819f6e3..828f584a 100644
--- a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.h
+++ b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererAML.h
@@ -41,7 +41,6 @@ class CRendererAML : public CLinuxRendererGLES
 
   // Feature support
   virtual bool Supports(EINTERLACEMETHOD method);
-  virtual bool Supports(EDEINTERLACEMODE mode);
   virtual bool Supports(ESCALINGMETHOD method);
   virtual bool Supports(ERENDERFEATURE feature);
 
diff --git a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.cpp b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.cpp
index 0fb0312..ba05557 100644
--- a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.cpp
+++ b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.cpp
@@ -71,11 +71,6 @@ bool CRendererOMX::Supports(EINTERLACEMETHOD method)
   return false;
 }
 
-bool CRendererOMX::Supports(EDEINTERLACEMODE mode)
-{
-  return false;
-}
-
 bool CRendererOMX::Supports(ESCALINGMETHOD mode)
 {
   return false;
diff --git a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.h b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.h
index 5f18452..6e84cb2 100644
--- a/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.h
+++ b/xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererOpenMax.h
@@ -38,7 +38,6 @@ class CRendererOMX : public CLinuxRendererGLES
 
   // Feature support
   virtual bool Supports(EINTERLACEMETHOD method);
-  virtual bool Supports(EDEINTERLACEMODE mode);
   virtual bool Supports(ESCALINGMETHOD method);
   
 
